util/job_manager.rb

```diff
@@ -38,7 +38,16 @@ def perform
       Job.current_job_id = self.uuid
       tick('Starting job') # Important to kill job before any code
 
+      puts "#{self.uuid} - Starting job... " + options['checksum']
```

Comment:
Wouldn't it be better if we use the log directly (so that the job id etc is automatic and conforms the syntax)?
```suggestion
      log "Starting job... #{options['checksum']}"
```